voxblox_skeleton/src/skeleton_generator.cpp

```diff
@@ -99,6 +98,8 @@ void SkeletonGenerator::generateSkeleton() {
   BlockIndexList blocks;
   esdf_layer_->getAllAllocatedBlocks(&blocks);
 
+  const size_t voxels_per_side = esdf_layer_->voxels_per_side();
```

Comment:
Can just store the voxels per side for the whole generator as a member; esdf and skeleton have the same size either way. :)